charts/alfresco-connector-ms365/templates/config-connector-ms365.yaml

```diff
@@ -0,0 +1,14 @@
+# Defines the properties required by the alfresco-connector-ms365 container
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "alfresco-connector-ms365.fullname" . }}-configmap
```

Comment:
if there are no other configmap I would just use the named template without `-configmap`, otherwise we still have the issue that the generated name may be longer than 64 chars

Reply:
agreed